app/code/Magento/CatalogImportExport/Model/Import/Product/Validator/Quantity.php

```diff
@@ -7,29 +7,28 @@
 
 use Magento\CatalogImportExport\Model\Import\Product\RowValidatorInterface;
 
-/**
- * Class Quantity
- */
 class Quantity extends AbstractImportValidator implements RowValidatorInterface
 {
     /**
-     * {@inheritdoc}
+     * @inheritdoc
      */
     public function isValid($value)
     {
         $this->_clearMessages();
         if (!empty($value['qty']) && (!is_numeric($value['qty'])
             && $value['qty'] !== $this->context->getEmptyAttributeValueConstant())
         ) {
-            $this->_addMessages(
-                [
-                    sprintf(
-                        $this->context->retrieveMessageTemplate(self::ERROR_INVALID_ATTRIBUTE_TYPE),
-                        'qty',
-                        'decimal'
-                    ),
-                ]
-            );
+            if ($this->context->retrieveMessageTemplate(self::ERROR_INVALID_ATTRIBUTE_TYPE) !== null) {
```

Comment:
Same as below. Please consider to enable call to original method
https://github.com/magento/magento2/blob/203a44f9e755fa6d2e057f1b99efbaff17546a80/app/code/Magento/CatalogImportExport/Test/Unit/Model/Import/Product/Validator/QuantityTest.php#L33

Reply:
please re-check the fix